tikv/mvcc/mvcc.go

```diff
@@ -59,6 +60,8 @@ type MvccLockHdr struct {
 	PrimaryLen     uint16
 	UseAsyncCommit bool
 	SecondaryNum   uint32
+	UseOnePC       bool
+	IsPessimistic  bool
```

Comment:
We don't need this field, we can check the Op instead.

Reply:
This happens when prewriting. Can the `Op` show whether the key is locked by a pessimistic lock? As I understood it, it only distinguishes the type of the modification like `Put`, `Delete`, `Insert`, `Lock`.

Reply:
No, the pessimistic lock has its own code `Op_PessimisticLock Op = 5`.